InstallCob4.sh

```diff
@@ -66,7 +66,7 @@ function BasicInstallation {
   wget http://packages.ros.org/ros.key -O - | sudo apt-key add -
   echo -e "\n${green}   INFO: Install ROS${NC}\n"
   sudo apt-get update
-  sudo apt-get install ros-indigo-rosbash python-ros-* ros-indigo-care-o-bot-robot -y --force-yes
```

Comment:
are you sure that this can be removed? this was needed to install rosdep (python-rosdep)